hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -576,7 +576,7 @@ message OMTokenProto {
     optional uint32 sequenceNumber = 7;
     optional uint32 masterKeyId    = 8;
     optional uint64 expiryDate     = 9;
-    required string omCertSerialId = 10;
+    optional string omCertSerialId = 10;
```

Comment:
it is already optional.

Reply:
Agree, this PR was created before that change. This will be fixed in next push after rebase.